cpp/turbodbc/Library/turbodbc/descriptions/string_description.h

```diff
@@ -9,6 +9,7 @@ namespace turbodbc {
  */
 class string_description : public description {
 public:
+	const static int multiplier = 4;
```

Comment:
`multiplier` does not communicate the purpose of this number well enough. I would suggest `max_code_units_per_unicode_code_point` is longer, but makes it clear that this is about Unicode and encoding (at least to those familiar with Unicode). See https://stackoverflow.com/a/27331885